companies/tasks/report_processing.py

```
@@ -0,0 +1,311 @@
+        Report.objects.filter(id=report_id).update(refined_content=refined_content)
+
+        data["refined_content"] = refined_content
+        return data
+    except Exception as e:
+        logger.error(f"보고서 정제 오류: {report_id} - {e}")
+        # Celery retry 호출 (지수 백오프)
+        raise self.retry(exc=e, countdown=60 * (2 ** self.request.retries))
+
+
+@shared_task(bind=True, max_retries=3)
+def extract_report_info_task(self, data: dict[str, Any]) -> dict[str, Any] | None:
```

Ruff (0.14.11)

Unused function argument: `self`

(ARG001)